### ISSUE TITLE
Question : Beta version 1

### ISSUE THREAD

**Issue by tgtje** [e107inc/e107#1165]
Is it possible to upgrade the Beta version1 download pack to a newer date(not a new version while Beta 2 is still a work in progress).

A lot of new users do have some issues when upgrading, in which many cases the solution often already is provided/build in the 'updated' present files..

**Comment by LaocheXe** [comment]
Yes it is, you use GitHub to download the current Files - upload them to your e107 dir. Then go into admin and you will see an update button - click it.

**Comment by tgtje** [comment]
Correct, but meant instead the download pack via 107.org ( = version 2 beta 1)  present pack is dated february) Although beta 2 is not at a finish point, the intermediate files might be updated in the beta 1 pack ( normally one wouldn't do in beta stages), but due to the interest hence question...

**Comment by Moc** [comment]
One can also simply enable the developer mode and then to go into Tools > Database > Sync with Github.

**Comment by tgtje** [comment]
Let me rephrase:  in example : i am new to e107 and start using the B1 pack, start and run into troubles > ask help > provided first solution > update with git files..(because a lot of old issues or already addressed to)..
So basically : if they start with B1 and run into some > issues arise (probably more than actually at present time ( > github files) exist. ( kind of needed? intermediate step > B1 works but present pre-b2 works better). (Posting as seen through 'beginners' eyes)

**Comment by LaocheXe** [comment]
So you want another version of e1072.x up on the site, pre beta 2 version correct?

**Comment by tgtje** [comment]
Indeed , but wanted is hard, more a point of view (not necessary for 
me), but for mostly new users (or updaters, but would require an upgrade 
version too).
But as posted before : when development is in stages; it would slightly 
step out of the normal staging route paths.

**Comment by Moc** [comment]
I see what you mean @tgtje. It may be useful for beginners as you say. Not sure what to do or recommend here. It may be confusing to have different versions (or 'builds' as it's called) of the same beta version. 

In the end, it's up to @CaMer0n to decide.

**Comment by Moc** [comment]
No longer relevant. Closing issue.
